CIP-????/README.md

```diff
@@ -202,7 +202,8 @@ Which results in the original content of the **msg** key:
 ## Rationale
 
 This design is simple, so many tools on the cardano blockchain can adopt it easily and a few have already started to implement it.
-The original CIP-0020 design allowed the addition of new entries like the `"enc":` key for encrypted messages in this CIP.
+The original CIP-0020 design allowed the addition of new entries like the `"enc":` key for encrypted messages in this CIP. Therefore the encoding format of the encrypted message was choosen to be UTF-8 instead of bytearrays, because it would break the backwards compatibility to CIP-0020. But maybe more important, it gives the user a simple text-format to handle such messages. Users can copy and paste the base64 encoded string(s) using there own tools for creation and verification. For example, a user can simply copy the encrypted format from an explorer and verify it with an external own local tool. Such messages are usally pretty short. Yes, the benefit of using bytearrays is to have less data (around -33% over base64), but the decision was made to sacrifice this benefit in favor of the base64 format for the reasons pointed out before.
```

Comment:
I'd argue that frontend application can still display the data as base64-encoded text string; while the on-chain data is itself being encoded as a plain byte array. Showing that data in a "human-readable" way is really a UI concern. As for backward compatibility; given that `enc` already specifies the behavior that a server needs to adopt in order to decode the `msg` payload; this may as well indicates that the data is specified as a sequence of bytes and not a sequence of UTF-8 text.